pkg/ecdsa/retry.go

```diff
@@ -0,0 +1,62 @@
+package ecdsa
+
+import (
+	"fmt"
+	"math/rand"
+	"sort"
+
+	"github.com/keep-network/keep-core/pkg/chain"
+)
+
+type byAddress []chain.Address
+
+func (ba byAddress) Len() int           { return len(ba) }
+func (ba byAddress) Swap(i, j int)      { ba[i], ba[j] = ba[j], ba[i] }
+func (ba byAddress) Less(i, j int) bool { return ba[i] < ba[j] }
+
+func EvaluateRetryParticipantsForSigning(
+	groupMembers []chain.Address,
+	seed int64,
+	retryCount uint,
+	quantity uint,
```

Comment:
Looks vague at the first sight, maybe we can rename it to, for example, `retryParticipantsCount`?

Reply:
[f5040e1](https://github.com/keep-network/keep-core/pull/3109/commits/f5040e1b63d7f44cf9ed88aeb7f08813c1a2ea20)